client/src/components/body/EducatorResources/ResourcesHeader.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+const ResourcesHeader = () => {
+  return (
+    <div
+      className={
+        "tw-w-full xs:tw-h-[30rem] md:tw-h-[30rem] lg:tw-h-[30rem] tw-bg-primary-blue tw-relative"
+      }
+    >
+      <div
+        className={
+          "tw-bg-primary-yellow tw-h-1/2 tw-w-[86.5%] tw-absolute tw-right-0 tw-top-[35%] tw-rounded-bl-2xl"
+        }
+      ></div>
+      <div
+        className={
+          "tw-bg-white tw-h-3/5 tw-w-[85%] tw-absolute tw-right-0 tw-top-[20%] tw-rounded-tl-lg tw-rounded-bl-lg"
+        }
+      >
+        <div
+          className={
+            "tw-flex tw-flex-col tw-text-left xs:tw-w-full md:lg:tw-w-7/12 tw-h-full tw-p-10  tw-justify-center"
+          }
+        >
+          <p
+            className={
+              "tw-font-poppins tw-font-bold xs:tw-text-[1.5rem] md:tw-text-3xl "
+            }
+          >
+            {" "}
+            Educator Resources{" "}
```

Comment:
The actual looks a bit different from the Figma. Yellow border is fully extended. The top blue gap is too thin. Buttons look slightly different. Font boldness is also a bit different

![image](https://github.com/user-attachments/assets/514b3ac7-d2a4-4f2d-8f64-2711b83c4357)

![image](https://github.com/user-attachments/assets/f53cb7b1-229d-4166-9974-5c38454ca117)

Reply:
Buttons should be yellow if they are active, otherwise, white. This is not the case when I was testing it.

Reply:
Updated and now in line with figma styling!